modules/fscloud/variables.tf

```diff
@@ -131,6 +131,11 @@ variable "bucket_configs" {
   }))
   description = "Cloud Object Storage bucket configurations"
   default     = []
+
+  validation {
+    condition     = length([for bucket_config in var.bucket_configs : true if contains([true], bucket_config.kms_encryption_enabled)]) == length(var.bucket_configs)
+    error_message = "fscloud mandates kms encryption."
```

Comment:
The FSCloud submodule mandates that `kms_encryption_enabled` is set to `true` for all buckets in `bucket_configs` input variable value